bootstrap/test/test-update.js

```diff
@@ -7,12 +7,14 @@
 // please check github issue number #242
 
 const assert = require('assert');
-const execSync = require('child_process').execSync;
+const path = require('path');
+const fs = require('fs');
 
 describe('Update Test', () => {
-  it('Is update was fine?', () => {
-	// FIXME(Wuseok): It will be updated new test module here.
-	// test-update.js will be check about npm update is successfully or not.
-	assert(true);
+  it('Is npm update working?', () => {
+	var pkg_stats = fs.statSync("package.json");

```

Comment:
I think you should do this test as follows:
* Test case 1. If the .pkg_timestamp doesn't exist, then .pkg_timestamp should be created after updating.
* Test case 2. If the .pkg_timestamp exists, then .pkg_timestamp should be modified (timestamp) after updating.